web/modules/custom/convert_text/src/ConvertText.php

```diff
@@ -34,18 +37,92 @@ protected static function convert(string $source_text, string $field_type): stri
 
       case 'html':
         $converter = new CommonMarkConverter();
-        return $converter->convert($source_text)->getContent();
+        $content = $converter->convert($source_text)->getContent();
+        return self::addLinkItMarkup($content);
 
       default:
         throw new \Exception("Invalid \$field_type of $field_type given");
+    }
+  }
+
+  /**
+   * Update local link tags with linkit data attributes.
+   */
+  protected static function addLinkItMarkup(string $source_text): string {
+    $dom = new \DOMDocument();
+    $dom->loadHTML($source_text);
+
+    foreach ($dom->getElementsByTagName('a') as $link) {
+      $href = $link->getAttribute('href');
+      if (!$href) {
+        continue;
+      }
+
+      $host = parse_url($href, PHP_URL_HOST) ?? '';
+
+      if (in_array($host, ['', 'digital.gov', 'www.digital.gov'])) {
+        $alias = parse_url($href, PHP_URL_PATH);
```

Comment:
$alias will be null when the url is `https://digital.gov` (note the lack of trailing slash). This will 500 the page.